packages/tc-ui/src/primitives/relationship/__tests__/e2e-create-relationships.cyp.js

```diff
@@ -23,6 +23,121 @@ describe('End-to-end - relationship creation', () => {
 		save();
 	});
 
+	describe('one-to-one relationship', () => {
+		it('can select a selection for one-to-one relationship', () => {
+			visitMainTypePage();
+			visitEditPage();
+
+			pickFavouriteChild();
+			save();
+
+			cy.url().should('contain', `/MainType/${code}`);
+			cy.get('#favouriteChild')
+				.should('have.text', `${code}-first-child`)
+				.should('have.attr', 'href', `/ChildType/${code}-first-child`);
+		});
+
+		it('can not select another selection for one-to-one relationship', () => {
+			visitMainTypePage();
+			visitEditPage();
+
+			pickFavouriteChild();
+			cy.get('#ul-favouriteChild span').should(
+				'have.text',
+				'e2e-demo-first-child',
+			);
+			cy.get('#favouriteChild-picker').should('be.disabled');
+			save();
+
+			cy.url().should('contain', `/MainType/${code}`);
+			cy.get('#favouriteChild')
+				.should('have.text', `${code}-first-child`)
+				.should('have.attr', 'href', `/ChildType/${code}-first-child`);
+		});
+	});
+
+	describe('one-to-many relationship', () => {
+		it('can create one-to-many relationship', () => {
+			visitMainTypePage();
+
+			visitEditPage();
+
+			// pick second child
+			pickChild();
+			save();
+
+			cy.url().should('contain', `/MainType/${code}`);
+			cy.get('#children>li')
+				.eq(0)
+				.should('have.text', `${code}-first-child`)
+				.find('a')
+				.should('have.attr', 'href', `/ChildType/${code}-first-child`);
+
+			cy.get('#children>li')
+				.eq(1)
+				.should('have.text', `${code}-second-child`)
+				.find('a')
+				.should('have.attr', 'href', `/ChildType/${code}-second-child`);
+		});
+
+		it('can select a selection for one-to-many relationship', () => {
+			visitMainTypePage();
+
+			visitEditPage();
+
+			// pick second child
+			pickChild();
+
+			cy.get('#ul-children')
+				.children()
+				.then(children => {
+					cy.wrap(children).should('have.length', 2);
+					cy.wrap(children)
+						.eq(0)
+						.find('span')
+						.should('have.text', 'e2e-demo-first-child');
+					cy.wrap(children)
+						.eq(1)
+						.find('span')
+						.should('have.text', 'e2e-demo-second-child');
+				});
+
+			save();
+
+			cy.url().should('contain', `/MainType/${code}`);
+			cy.get('#children>li').then(children => {
+				cy.wrap(children)
+					.eq(0)
+					.should('have.text', `${code}-first-child`)
+					.find('a')
+					.should(
+						'have.attr',
+						'href',
+						`/ChildType/${code}-first-child`,
+					);
+				cy.wrap(children)
+					.eq(1)
+					.should('have.text', `${code}-second-child`)
+					.find('a')
+					.should(
+						'have.attr',
+						'href',
+						`/ChildType/${code}-second-child`,
+					);
+			});
+		});
+
+		it('can select another selection for one-to-many relationship', () => {
```

Comment:
```suggestion
		it('does not disable when relationship already exists', () => {
```
This test shoudl move before the previous one too I think